pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/answer/DecimalAnswer.java

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.ddp.model.activity.instance.answer;
+
+import com.google.gson.annotations.SerializedName;
+import org.broadinstitute.ddp.model.activity.types.QuestionType;
+
+import java.math.BigDecimal;
+
+public final class DecimalAnswer extends Answer<BigDecimal> {
+    @SerializedName("value")
+    private BigDecimal value;
```

Comment:
What type is this going to be serialized to, or is this going to be updated in your next PR with the deserialization updates?

Reply:
Will be replaced with DecimalDef